.ci/atime/tests.R

```diff
@@ -300,8 +300,8 @@ test.list <- atime::atime_test_list(
       data.table::tables(env = test_env, silent = TRUE, index = TRUE)
       NULL
     },
-    before = "2b191aec3df51675a7ab5e6701384a8b89470af6",  # The merge-base parent commit for PR #2606
-    after =  "66024e6ea5304ae059b18c71de9973715ddb77ce"),   # The final commit of PR #2606
+    before = "2b191aec3df51675a7ab5e6701384a8b89470af6",  # Merge-base parent commit for PR #2606 (https://github.com/Rdatatable/data.table/commit/2b191aec3df51675a7ab5e6701384a8b89470af6
```

Comment:
please read https://github.com/Rdatatable/data.table/wiki/Performance-testing#documentation-of-historical-commits
the link should be to a PR page (on a page where we can see how the commit id is related to the PR), not to the commit page (on which we do not see anything related to PR2606).

also there is no PR2606, but there is an issue 2606.